frontend/src/app/user/my-submissions/my-submissions.component.ts

```diff
@@ -0,0 +1,44 @@
+import {ChangeDetectionStrategy, Component, OnInit, ViewChild} from '@angular/core';
+import {Submission} from "../shared/submission.model";
+import {SubmissionService} from "../shared/submission.service";
+import {MatSort, MatSortable} from "@angular/material/sort";
+import {MatTableDataSource} from "@angular/material/table";
+import {MatDialog} from "@angular/material/dialog";
+import {RecommendationDialogComponent} from "./recommendation-dialog/recommendation-dialog.component";
```

Comment:
please enable TSLint in intellij under language & frameworks > Typescript > Automatic tsLint config. and fix all the TSLint errors